.github/workflows/pull-upstream.yml

```
@@ -33,7 +33,7 @@ jobs:
         .github/scripts/check-whitespace.sh
 
     - name: Create Pull Request
-      uses: peter-evans/create-pull-request@v4
+      uses: peter-evans/create-pull-request@v4.2.3
```

```suggestion
      uses: peter-evans/create-pull-request@v4
```
There's no need for changes here